cpeparse/rpmname.go

```diff
@@ -66,23 +66,22 @@ func FieldsFromRPMName(s string) (name, ver, rel, arch string, err error) {
 }
 
 // FromRPMName parses CPE name from RPM package name
-func FromRPMName(s string) (*wfn.Attributes, error) {
+func FromRPMName(attr *wfn.Attributes, s string) error {
```

Comment:
I think that this function should still have the same signature.
Create attributes, set Vendor to any because it's a part of this logic, and then return it.
But not going to block.